tests/h/search/client_test.py

```diff
@@ -34,87 +30,41 @@ def test_conn_is_read_only(self):
         with pytest.raises(AttributeError, match="can't set attribute"):
             client.conn = "changed"
 
-    def test_defaults_to_es1(self):
+    def test_defaults_to_es6(self):
```

Comment:
This test name makes less sense when there is only one Elasticsearch instance. I think it would make sense to have one test for the version property and one for the conn property (note there is already a `test_it_sets_the_index_and_conn_properties` test above, but that doesn't check anything beyond the fact that `conn` is not None!)

Reply:
I made a test for each property now and removed the default test.